docker/docker-image.sh

```diff
@@ -1,7 +1,12 @@
 #!/bin/bash
 
-if [ x"${TRAVIS_SMALLTALK_VERSION}" != x"Pharo-4.0" ]; then
-    exit 0
+set -u
+
+if [ x"${TRAVIS_SMALLTALK_VERSION}" = x"Pharo-4.0" ]; then
```

Comment:
What about making this symmetric and have Pharo-4.0 and Pharo-5.0 behave the same? Copy the VM in both cases. Could you then fold it into a single Dockerfile as well?